public/css/game.css

```diff
@@ -40,6 +40,7 @@
   width: 20vh;
   height: 20vh;
   margin: 1vh;
+  position: relative;
```

Comment:
Why did you have to include this?

Reply:
Z-index requires position relative to work.